app/src/main/java/br/com/concrete/tentacle/features/library/loan/LoanViewHolder.kt

```diff
@@ -42,13 +38,10 @@ LoanViewHolder(item: View) : RecyclerView.ViewHolder(item) {
             )
 
             item.setOnClickListener {
-                item.context?.startActivity(
-                    Intent(
-                        Intent.ACTION_VIEW, Uri.parse(
-                            Utils.assembleUrlYouTube(video.video_id)
-                        )
-                    )
-                )
+                val extras = Bundle()
+                extras.putString(PinchToZoomActivity.ID, video.video_id)
+                extras.putString(PinchToZoomActivity.TYPE, PinchToZoomActivity.VIDEO)
+                startPinchActivity(item.context as AppCompatActivity, extras)
```

Comment:
É interessante pensar aqui no safe cast. 
Se o contexto não for de uma AppCompatActivity esse cast vai lancar uma exception. Tudo bem que especificamente neste contexto dificilmente isso iria acontecer. Mas não custa previnir

Da  uma sacada https://kotlinlang.org/docs/reference/typecasts.html#safe-nullable-cast-operator

Uma abordagem segura é usar o safe cast e se der sucesso chamar a activity usando a extension como vc fez ou se der nulo usar uma extension de context sem animação por exemplo. Sacou? Só para previnir mesmo